contract/src/main/java/io/neow3j/contract/NeoToken.java

```diff
@@ -185,7 +188,7 @@ public List<ECPublicKey> getCommittee() throws IOException {
      * <p>
      * The vote count is based on the summed up NEO balances of the respective candidate's voters.
      *
-     * @return the candidate public keys and their corresponding vote count.
+     * @return the validator public keys and their corresponding vote count.
```

Comment:
I don't agree with the word change here. The list returned by the neo-node contains candidates and the current committee members. They are not necessarily validators.
Additionally, in the "worst case" this returns 256 candidates.

Reply:
You're totally right. I'll change it back.